Content.Trauma.Shared/Chaplain/Components/IgniteOnHolyDamageComponent.cs

```diff
@@ -0,0 +1,14 @@
+using Content.Goobstation.Maths.FixedPoint;
+
+namespace Content.Trauma.Shared.Chaplain.Components;
+
+[RegisterComponent]
+public sealed partial class IgniteOnHolyDamageComponent : Component
+{
+    [DataField("fireStacks")]
+    public float FireStacks = 1f;
+
+    // The minimum amount of damage taken to apply fire stacks
+    [DataField("threshold")]
+    public FixedPoint2 Threshold = 15;
```

Comment:
Networked, `[DataField]`, actually document it not just a single comment use xmldoc summary